### ISSUE TITLE
Help!!! Laravel 8 Error - Required parameter $imageUrl follows optional parameter $title

### ISSUE THREAD

**Issue by devhoussam** [realrashid/sweet-alert#113]
Hi I'm implement **sweet-alert** with **laravel 8 version**  when I try to redirect with success message it display this error message it contain : 

![Image of the problem message](https://i.imgur.com/pmzxdCD.png)

`Required parameter $imageUrl follows optional parameter $title
`

**My Code :** 

```
<?php

...

use RealRashid\SweetAlert\Facades\Alert;

class RegisteredUserController extends Controller
{

	...
	
    public function store(Request $request)
    {
        $request->validate([
            'name' => 'required|string|max:255',
            'email' => 'required|string|email|max:255|unique:users',
            'password' => 'required|string|confirmed|min:8',
        ]);

        Auth::login($user = User::create([
			'name' => $request->name,
            'email' => $request->email,
            'password' => Hash::make($request->password),
        ]));

        event(new Registered($user));
		
		return redirect(RouteServiceProvider::HOME)->with('success', 'Amazing! Your Account has been Created Successfully!');
    }
}

```

**Comment by realrashid** [comment]
Duplicate of #108

**Comment by realrashid** [comment]
Fix in v3.2.1
